src/encoded/static/components/item-pages/UserView.js

```diff
@@ -767,3 +768,26 @@ export class ImpersonateUserForm extends React.PureComponent {
     }
 
 }
+const UserPageTitle = React.memo(function UserPageTitle(props) {
+    const { alerts, context, session, href, schemas } = props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['schemas' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4798956)